src/Forms/FormField.php

```diff
@@ -384,7 +391,7 @@ public function Link($action = null)
             );
         }
 
-        $link = Controller::join_links($this->form->FormAction(), 'field/' . $this->name, $action);
+        $link = Controller::join_links($this->form->FormAction(), 'field/' . $this->getName(), $action);
```

Comment:
The `name` property is `null` for `CompositeField` because it builds its name at runtime based on the child field names. We have to call `getName()` to get a valid URL here.